common.sh

```diff
@@ -250,10 +250,10 @@ add_postupgrade()
     local postupgrade_name
 
     script_name=$1
-    postupgrade_name="$PREUPG_POSTUPGRADE_DIR/`basename $script_name`"
+    postupgrade_name="$PREUPG_POSTUPGRADE_DIR/$(basename $script_name)"
     [ ! -f "$script_name" ] && log_error "$script_name does not exist." && exit_error
     [ -f "$postupgrade_name" ] && log_warning "$script_name already exists in $PREUPG_POSTUPGRADE_DIR"
-    cp -af "$script_name" "$postupgrade_name"
+    cp -af ${script_name} ${$postupgrade_name}
```

Comment:
The rightmost argument is apparently a typo.  But why are you removing quotes from the `"$script_name"`? The same for other places in this commit.

All in all, I believe this commit does not "fix" or improve the code, rather the opposite.